modin/pandas/series.py

```diff
@@ -152,14 +201,27 @@ def __rand__(self, other):
         new_self, new_other = self._prepare_inter_op(other)
         return super(Series, new_self).__rand__(new_other)
 
-    def __array__(self, dtype=None):
+    def __array__(self, dtype=None):  # noqa: PR01, RT01
```

Comment:
why don't you document the method instead of squelching the warning?

Reply:
Dunder methods were documented too.